Source/AudioSettingsDialog.cpp

```
@@ -64,6 +64,30 @@ AudioSettingsDialog::AudioSettingsDialog(Audio::AudioDeviceManager* audioManager
     , m_isRenderingToCache(false)
     , m_activeTab("settings")
 {
+    // Subscribe to stream errors (e.g. device disconnection)
+    if (m_audioManager) {
```

Clang (14.0.6)

[warning] implicit conversion 'Audio::AudioDeviceManager *' -> bool

(readability-implicit-bool-conversion)